Makefile

```diff
@@ -0,0 +1,27 @@
+MSBUILD = xbuild
+MDOC = mdoc/bin/Release/mdoc.exe
+
+all: prepare build
+
+build: $(MDOC)
+$(MDOC): external/SharpZipLib/bin/Release/ICSharpCode.SharpZipLib.dll external/Mono.Cecil/bin/net_4_0_Release/Mono.Cecil.dll
+	$(MSBUILD) apidoctools.sln /p:Configuration=Release
+
+prepare:
+	git submodule update --init --recursive
+
+external/SharpZipLib/bin/Release/ICSharpCode.SharpZipLib.dll:
+	$(MSBUILD) external/SharpZipLib/ICSharpCode.SharpZipLib.NET45/ICSharpCode.SharpZipLib.csproj /p:Configuration=Release
+
+external/Mono.Cecil/bin/net_4_0_Release/Mono.Cecil.dll:
+	$(MSBUILD) external/Mono.Cecil/Mono.Cecil.sln /p:Configuration=net_4_5_Release
+
+clean:
+	$(MSBUILD) apidoctools.sln /t:clean
+	$(MSBUILD) external/SharpZipLib/ICSharpCode.SharpZipLib.NET45/ICSharpCode.SharpZipLib.csproj /t:clean
+	$(MSBUILD) external/Mono.Cecil/Mono.Cecil.sln /t:clean
+
+check: check-mdoc
+
+check-mdoc:
+	cd mdoc; $(MAKE) check
```

Comment:
Should we just merge the contents of `mdoc/Makefile` into this file, and delete `mdoc/Makefile`?